### DISCUSSION TITLE
Installing the extension on existing Chrome window?

### DISCUSSION THREAD

**Discussion by artt** [wxt-dev/wxt#1314] [Q&A] [answered]
I'm writing a tab manager extension (b/c I found myself with too many tabs all the time). I'm currently using Vite + React. I had to build and manually reload the extension every time I wanted to test it before I learned about WXT.

But I soon found out that WXT opens up an new instance of a browser. I'm wondering if there's a way to get WXT to run the extension on existing windows, since that's where my tabs are, and I need lots of tabs (from actual usage) to test my extension. Having looked at [this answer](https://github.com/wxt-dev/wxt/discussions/922#discussioncomment-10420335), I assume the answer is no... but asking just in case. Or if there's a better way than what I'm doing now.

Thanks!

**Comment by aklinker1** [comment, marked as answer]
Yeah, answer is no. `web-ext` doesn't support that. Loading it manually into your main profile once should work the same as if you opened a browser though.

**Reply by artt** [reply]
Ah. This is great! So you can just load unpacked once (from `[rootdir]/.output/chrome-mv3`) and everything works. Thanks!
